tests/man.test

```diff
@@ -814,8 +814,8 @@ true
 [null, 1]
 
 [.[] | tonumber?]
-["1", "invalid", "3", 4]
-[1, 3, 4]
+["1", "invalid", "3", 4, true, false]
+[1, 3, 4, 1, 0]
```

Comment:
I'm just a random passerby in this repo, but having spent an obscene amount of time trying (failing) to get doc-building working yesterday (on Windows, long story), I think what you need to do here is make the change to [docs/content/manual/manual.yml:2539](https://github.com/davidfetter/jq/blob/4698694e37920d13409e8394237a4b0d0d25b254/docs/content/manual/manual.yml#L2539-L2540) and then rebuild the docs per [docs/README.md](/davidfetter/jq/blob/4698694e37920d13409e8394237a4b0d0d25b254/docs/README.md).

Maybe some words and another example in the [`tonumber` section of the manpage](/davidfetter/jq/blob/4698694e37920d13409e8394237a4b0d0d25b254/docs/content/manual/manual.yml#L1454-L1464) might be nice, too?

Reply:
Thanks! Getting that patch modified as you've suggested.